### PR TITLE
retirement of compara gene families

### PR DESCRIPTION
**Pull request by thiagogenez**
The general Gene Family documentation on the Ensembl website does not mention the retirement of the Gene Families. This PF attempts to reflect this retirement. In particular, the following changes should be considered for updates in the [public-plugins](https://github.com/Ensembl/public-plugins) repo:

- docs/htdocs/info/docs/api/compara/compara_tutorial.html: references to Compara Gene Families should be removed.
- ensembl/htdocs/info/genome/compara/family.html: a notice should be added to inform the user that Gene Families have been retired
- ensembl/htdocs/info/genome/compara/family_mcl.html.bak: this file should be removed as it is no longer relevant.

[1] https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-5357

### CODE REVIEW

File: docs/htdocs/info/docs/api/compara/compara_tutorial.html [hunk 1]
```diff
@@ -369,10 +367,10 @@ <h3>Print coordinates</h3>
 <h2 id="gene_resources">Gene trees, Homologies and Protein clusters</h2>
 
 <p>
-All the gene trees, homologies and families refer to GeneMembers and SeqMembers. 
+All the gene trees and homologies refer to GeneMembers and SeqMembers. 
 Homology objects store orthologous and paralogous relationships between members, 
-GeneTree objects represent the evolutionary history of a set of members, 
-and Family objects are clusters of members.
+GeneTree objects represent the evolutionary history of a set of members 
+are clusters of members.
```
Comment (twalsh-ebi) [inline docs/htdocs/info/docs/api/compara/compara_tutorial.html:373]:
```suggestion
GeneTree objects represent the evolutionary history of a set of members.
```

File: ensembl/htdocs/info/genome/compara/family.html [hunk 2]
```diff
@@ -4,6 +4,12 @@
 </head>
 
 <body>
+<dialog open>
+<h2>Compara gene and protein families were retired as of release 102, as announced in a <a href="https://www.ensembl.info/2020/11/24/gene-and-protein-families-retiring/">November 2020 Ensembl blog post</a>.</h2>
+  <form method="dialog">
+  <center><button>Close</button></center>
+  </form>
+</dialog>
```
Comment (twalsh-ebi) [inline ensembl/htdocs/info/genome/compara/family.html:12]: This presents the message loud and clear. But I was wondering, could it be made more consistent with other Ensembl web code in some way, such as by using a `div`-based `info` box like [the one describing the taxonomic group(s) for which Orthology QC is done](https://github.com/thiagogenez/public-plugins/blob/7a5cb7967fa03a4602f92c701bb33add4dacaa88/ensembl/htdocs/info/genome/compara/Ortholog_qc_manual.html#L47-L54) (which can be seen on the [Orthology QC webpage](https://jul2022.archive.ensembl.org/info/genome/compara/Ortholog_qc_manual.html))?

**Review by twalsh-ebi — COMMENTED** [review]
For what it's worth, this looks good to me from a Compara perspective. I just have a couple of comments.

File: ensembl/htdocs/info/genome/compara/family.html (hunk 2 above)
Comment (ens-ap5) [reply ensembl/htdocs/info/genome/compara/family.html:12]: i agree with Thomas - this is not Ensembl-like HTML, and goes against our style gudelines for the site. We also don't really want to be able to dismiss the message - if this page getting cached, another user might not see it.

Please change this to use the standard "info" class on a div element.

Comment (thiagogenez) [reply ensembl/htdocs/info/genome/compara/family.html]: Thanks, @twalsh-ebi and @ens-ap5 for highlighting this issue. Fixed!

**Review by twalsh-ebi — APPROVED** [review]

**Pull request merged**